dask_saturn/core.py

```diff
@@ -9,7 +10,13 @@
 from .backoff import ExpBackoff
 
 SATURN_TOKEN = os.environ.get("SATURN_TOKEN", "")
+if SATURN_TOKEN == "":
+    warnings.warn("Environment variable SATURN_TOKEN not set", RuntimeWarning)
+
 BASE_URL = os.environ.get("BASE_URL", "")
+if BASE_URL == "":
+    warnings.warn("Environment variable BASE_URL not set", RuntimeWarning)
+
 HEADERS = {"Authorization": f"token {SATURN_TOKEN}"}
 DEFAULT_WAIT_TIMEOUT_SECONDS = 1200
```

Comment:
I like the proposal from your comment. We should also get rid of passing cluster_url directly into the constructor.

In terms of wording of the error message, it is generally good practice to suggest a solution in addition to notifying the user of what happened.

Reply:
ok how bout this? https://github.com/saturncloud/dask-saturn/pull/17/commits/b688d179c1341297962049ebba6bdf8754213e87

I don't understand the implications of not passing in `cluster_url` so I think that change should be a separate PR

Reply:
Ok that is fine.